File: Day 1/detect_capital.py
def detectCapitalUse(self, word: str) -> bool:
        caps = 0
        for c in word:
            if c >='A' and c <= 'Z':
                caps = caps + 1
        if caps == 0 or caps == len(word):
            return True
        return caps == 1 and word[0].isupper()

File: Day 1/DetectCapital.java
package august2020;

public class DetectCapital {
	
	public boolean detectCapitalUse(String word) {
        int capsCount = 0;
        for(int i = 0; i < word.length(); i++){
            if(Character.isUpperCase(word.charAt(i)))
                capsCount++;
        }
        if(capsCount == word.length() || capsCount == 0) return true;
        return capsCount == 1 && Character.isUpperCase(word.charAt(0));
    }

}
